src/sentry/api/endpoints/organization_details.py

```diff
@@ -141,6 +144,14 @@ def validate_safeFields(self, attrs, source):
             raise serializers.ValidationError('Empty values are not allowed.')
         return attrs
 
+    def validate_attachmentsRole(self, attrs, source):
+        value = attrs[source]
+        try:
+            roles.get(value)
+        except KeyError:
+            raise serializers.ValidationError('Invalid role')
```

Comment:
Should we tell the user what the valid states are?

Reply:
This should never happen anyways unless someone bypasses the UI.